go.mod

```diff
@@ -31,10 +31,10 @@ require (
 	github.com/decred/dcrd/dcrutil/v3 v3.0.0
 	github.com/decred/dcrd/dcrutil/v4 v4.0.1
 	github.com/decred/dcrd/hdkeychain/v3 v3.1.1
-	github.com/decred/dcrd/rpc/jsonrpc/types/v3 v3.0.0
+	github.com/decred/dcrd/rpc/jsonrpc/types/v4 v4.0.0
 	github.com/decred/dcrd/txscript/v4 v4.1.0
 	github.com/decred/dcrd/wire v1.6.0
-	github.com/decred/dcrdata/v7 v7.0.0-20211216152310-365c9dc820eb
+	github.com/decred/dcrdata/v8 v8.0.0-20230617164141-fa4d8e1b4e8e
```

Comment:
`v8.0.0-20230627030434-e7e8ab240260` this is the commit tied to the latest commit on master not `v8.0.0-20230617164141-fa4d8e1b4e8e`. Run delete the dcrdata module entry on go.mod and run `go mod tidy` it should resolve to the latest commit.

Reply:
https://github.com/crypto-power/cryptopower/issues/26 we aren't trying to target the latest master commit

Reply:
Any specific reason why we are freezing on `fa4d8e1b4e8e`.

Reply:
yes, because that is the v1.8 upgrade which we need, until an official release is done

Reply:
Makes sense. I see upstream have not made a release for some time.